162-plot-dragging/app.R

```diff
@@ -0,0 +1,36 @@
+shinyApp(
```

Comment:
I think we want to assume this script can be run standalone, so add `library(shiny)`

Reply:
Also, for good measure, should we be testing `imageOutput()` as well?

Reply:
There's many apps in this repo that don't have `library(shiny)` -- calling `shiny::runApp()` automatically attaches shiny.

I'll add `imageOutput` examples.